srcpkgs/midori/template

```diff
@@ -1,31 +1,18 @@
 # Template file for 'midori'
 pkgname=midori
-version=0.5.11
-revision=2
-lib32disabled=yes
-build_options="zeitgeist"
+version=6.0
+revision=1
 build_style=cmake
-configure_args="-DUSE_GTK3=1 -DHALF_BRO_INCOM_WEBKIT2=ON
- $(vopt_if zeitgeist '-DUSE_ZEITGEIST=1' '-DUSE_ZEITGEIST=0')"
-hostmakedepends="pkg-config intltool librsvg-utils glib-devel vala-devel libzeitgeist-devel"
-makedepends="gtk+3-devel libxml2-devel vala-devel $(vopt_if zeitgeist libzeitgeist-devel)
- dbus-glib-devel librsvg-devel libnotify-devel libsoup-gnome-devel
- webkit2gtk-devel sqlite-devel libunique-devel libXScrnSaver-devel gcr-devel"
+hostmakedepends="glib-devel gobject-introspection intltool librsvg-utils
+pkg-config vala-devel"
+makedepends="gcr-devel libpeas-devel librsvg-devel libsoup-gnome-devel
+vala-devel webkit2gtk-devel"
```

Comment:
This also depends on gtk-devel and sqlite-devel - but it's pulled in anyway, so I guess it doesn't matter...

Reply:
I think  so